### ISSUE TITLE
Add URL to feature demos/samples

### ISSUE THREAD

**Issue by marco-c** [mozilla/platatus#88]

**Comment by digitarald** [comment]
Should this allow a list? (I'd vote yes)

Needs to be done for each feature. Also currently the UI doesn't show a sample URL.

**Comment by brendandahl** [comment]
The feature list is starting to feel a bit crowded. How much stuff do we want to show? Couldn't samples/demo be discovered through mdn?

**Comment by digitarald** [comment]
Indeed, it doesn't feel as important. We should explore a way to add additional hierarchy and granularity in the information architecture.
